standalone_self_upgrade.cf.in

```diff
@@ -115,11 +115,38 @@ bundle agent cfengine_software
 # @brief Variables to control the specifics in desired package selection
 {
   vars:
+    any::
+
+      # Extract the hub binary version info if it's available. Only expected to
```

Comment:
shame you couldn't re-use the same bits here from update.cf.in, but I understand why. 👍

Reply:
I know, painful.